README.md

```diff
@@ -8,7 +8,7 @@
 
 > Tracing supported: 
 - [LTTng](https://lttng.org) (Kernel CPU scheduling, Processes, Threads, Block IO/Disk, Syscalls, File events, etc)
-- [perf](https://perf.wiki.kernel.org/) CPU Sampling(cpu-clock)
+- [perf](https://perf.wiki.kernel.org/) (Generic events)
```

Comment:
Please leave CPU Sampling (cpu-clock) as-is as that is supported by PerfDataTxtExtension. But yes, add Generic Events. No need to differentiate between plugins here, just in the capture readme